src/com/moneydance/modules/features/securityquoteload/GetYahooHistQuote.java

```diff
@@ -63,10 +64,12 @@
 	private String currencyID="";
 	public GetYahooHistQuote(String tickerp, QuoteListener listenerp, CloseableHttpClient httpClientp,String tickerTypep, String tidp) {
 		super(tickerp, listenerp, httpClientp, tickerTypep,  tidp);
+		String convTicker = ticker.replace("^", "%5E");

```

Comment:
I noticed URLEncoder is imported but not used, even though it seems like you're doing a manual encoding of the ^ sign here.  Did you mean to use URLEncoder.encode() do encode the entire ticker symbol?